src/tcgwars/logic/impl/gen4/RisingRivals.groovy

```diff
@@ -1660,15 +2190,19 @@ public enum RisingRivals implements LogicCardInfo {
             energyCost C
             attackRequirement {}
             onAttack {
-              damage 0
+              damage 10
+              whirlwind()
```

Comment:
I think whirlwind is actually backwards here. Which Pokémon gets switched with which is pretty important.

Reply:
Actually, it seems that WotC cards may have had it backwards? Whirlwind should definitely be double checked that it works right then. Might want to confirm with Jason if this order matters in WotC formats.